commands/MessageCollector.go

```diff
@@ -0,0 +1,42 @@
+package commands
+
+import (
+	"errors"
+	"github.com/bwmarrin/discordgo"
+	"strconv"
+)
+
+/**
+* /collect <message-snowflake-id> <condition-type> <condition-value>
```

Comment:
Don't add extra formatting like banners or stars (see [here](https://golang.org/doc/effective_go#commentary)). I would just do multiline comments with `//` in front of each line. Also I would add a brief description as the first line and then usage after.